packit/config.py

```diff
@@ -130,6 +131,7 @@ def get_from_dict(cls, raw_dict: dict, validate=True) -> "Config":
         config.github_app_id = raw_dict.get("github_app_id", "")
         config.github_app_cert_path = raw_dict.get("github_app_cert_path", "")
         config.webhook_secret = raw_dict.get("webhook_secret", "")
+        config.openshift = raw_dict.get("openshift", "cli")
```

Comment:
this needs a better name: openshift is a name of a project and doesn't really tell what the config value means

Reply:
`command_runner`, `actions_runner`, `actions_environment`?
@jpopelka You are the expert to set proper variables. Any idea?

Reply:
I have selected `actions_environment`, because of `run_command` is called either in CLI or in Openshift and there are environments.